docs/tutorials/FlowTutorial.md

```diff
@@ -231,6 +231,16 @@ make DESIGN_CONFIG=./designs/sky130hd/ibex/config.mk
 As the flow executes, check out the ORFS directory contents and their
 significance.
 
+If you started running a design and the terminal in which you initiated the run had some errors for some reason, you may want to delete all partially generated files and start a fresh run(`ibex` in this case). You can accomplish this task by:
+
+```
+make clean_all DESIGN_CONFIG=./designs/sky130hd/ibex/config.mk
+```
+
+You can also delete files related to individual stages of RTL to GDSII conversion like synthesis, floorplanning, macro placement, clock-tree synthesis,  routing and layout generation `clear_all` by `clean_synth`, `clean_floorplan`, `clean_place`, `clean_cts`, `clean_route`, `clean_finish` respectively.
+
+ORFS can generally restart from a previous partial run, so in that case these clear instructions can be ignored.
+
```

Comment:
```suggestion
ORFS can generally restart from a previous partial run.  If you have errors which prevent restarting for some reason, you can delete all partially generated files and start a fresh run(`ibex` in this case).  This could happen if the tool crashed or was killed in the middle of writing a file.  You can accomplish this task by:

```
make clean_all DESIGN_CONFIG=./designs/sky130hd/ibex/config.mk
```

You can also delete files related to individual stages of RTL to GDSII conversion like synthesis, floorplanning, macro placement, clock-tree synthesis,  routing and layout generation `clear_all` by `clean_synth`, `clean_floorplan`, `clean_place`, `clean_cts`, `clean_route`, `clean_finish` respectively.
```